profiler/src/profiler/TracyTimelineItemThread.cpp

```diff
@@ -144,7 +144,7 @@ int64_t TimelineItemThread::RangeEnd() const
 
 void TimelineItemThread::HeaderTooltip( const char* label ) const
 {
-    m_view.HighlightThread( m_thread->id );
+    m_view.HighlightThread(m_thread->id, false);
```

Comment:
Please use the correct code style across the changes.

Reply:
Done, sorry !